support/remote_health_monitoring/opting-out-of-remote-health-reporting.adoc

```diff
@@ -13,11 +13,7 @@ ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
 You may choose to opt out of reporting health and usage data for your cluster.
 endif::[]
 
-ifdef::openshift-dedicated[]
-On {product-title}, remote health reporting is always enabled. You cannot opt out.
-
-// TODO: Is there even a reason to include this assembly for OSD? Could you remove it, and just mention in the other assemblies that you can't opt out?
-endif::[]
+// moved OSD note on not able to opt out to about telemetery
```

Comment:
Moved to: https://github.com/openshift/openshift-docs/pull/63305/files#diff-a380b4a4a3b7cdf4ef40cbe2d4504fba42212a1ba48b3ec9a700eeec15f0ffa0R37